docs/Contributing.adoc

```diff
@@ -6,11 +6,38 @@ Herein lies the contribution guidelines for helping out with this project. Do ta
 
 === General rules
 
-* The Ansible <<coc,Code of Conduct>> still applies.
-* For git messages, branch names, ..., follow <<gitstyle,Git Style Guide>>
-* Should you wish to work on a completely new standard, GREAT, but please file an issue for tracking.
+* The Ansible link:https://docs.ansible.com/ansible/latest/community/code_of_conduct.html[Code of Conduct] still applies.
+* For git messages, branch names, ..., follow link:https://github.com/agis/git-style-guide[Git Style Guide].
+* Should you wish to work on a completely new standard, GREAT, but please file an **issue** for tracking. In your Pull Request, specify `closes #NUM` to automatically close the issue when the PR is merged.
 * To contribute, fork and make a pull request against the `development` branch.
-* All tasks should be in YAML literal.
+*  Use link:https://asciidoctor.org/docs/asciidoc-writers-guide/[asciidoc] for documentation.
+* Pull requests should only change one role, one config, or one role as applied to one config. There may be exceptions to this rule, but you must explain why a special exception should apply to a PR. Ask yourself, « Can this pull request be separeted in several smaller pull requests ? »
+* Pull Request titles:
+** Must start with `Add | Change | Fix | HOTFIX | Remove`
```

Comment:
What is the difference between a Fix and a HOTFIX?

Reply:
i give an example later in the doc: If a production catalog item is broken because, let's say, some Images disappeared from the cloud-provider, we need a way to identify that it's a HOTFIX for production.

HOTFIX is to identify a small change developed to correct a major bug or fault and to be released as quickly as possible. We could also put URGENT in the title.

I thought it was self-explanatory but now you make me doubt :)

Reply:
Normally what I see for a HOTFIX workflow involves creating a fork of the repository from the previous release and then fixing the issue in the fork. So a FIX follows trunk-based development but a HOTFIX is not merged into the trunk.

Reply:
"HOTFIXes" are for old, cold releases.

Reply:
i'm removing HOTFIX